frontend/src/lib/lemon-ui/LemonTextArea/LemonTextArea.tsx

```diff
@@ -13,6 +13,9 @@ import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
 import { Link } from 'lib/lemon-ui/Link'
 import { Tooltip } from 'lib/lemon-ui/Tooltip'
 import { LemonTabs } from '../LemonTabs'
+import image_blob_reduce from 'image-blob-reduce'
+
+const reduce = image_blob_reduce()
```

Comment:
In general this is nice... but this adds [50KB of JS](https://bundlephobia.com/package/image-blob-reduce@4.1.0) to every lemon textarea. Is there any way we can lazy load this (`await import('image-blob-reduce')`) when needed?

Reply:
that's actually pretty cool 🙌 

<img width="1063" alt="Screenshot 2023-03-23 at 11 31 21" src="https://user-images.githubusercontent.com/984817/227191687-4e5aac11-2096-47aa-86d1-e532265b4b4a.png">

Here's the library being imported on first use 🚀